### ISSUE TITLE
Labs problem with white balance in RAW Timelapse

### ISSUE THREAD

**Issue by MrProst** [gopro/labs#349]
When recording a timelapse in RAW format, after about 100-150 frames of recording, the white balance changes and a vignette effect appears.
This happens every time a timelapse is recorded.
![G0025593](https://github.com/gopro/labs/assets/133163316/8cb3425c-8e49-4705-8ac0-43c26cfa8c35)
![G0025592](https://github.com/gopro/labs/assets/133163316/ee06e96c-26f0-4229-abbf-f26470d58be0)

**Comment by dnewman-gpsw** [comment]
Yes, that is normal for Raw.  JPEG has all the processing, Raw does not.  White balance in Raw is just metadata, it is up to the Raw development tool to do the white balance corrections.   The flat fielding (corner vignetting correction) is also a postprocess.
